WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -930,6 +936,17 @@ private void showNewPostsBar() {
         AniUtils.startAnimation(mNewPostsBar, R.anim.reader_top_bar_in);
         mNewPostsBar.setVisibility(View.VISIBLE);
 
+        // hide the bar when the recycler is scrolled
+        mRecyclerView.addOnScrollListener(new RecyclerView.OnScrollListener() {
+            @Override
+            public void onScrolled(RecyclerView recyclerView, int dx, int dy) {
+                super.onScrolled(recyclerView, dx, dy);
+                if (dy >= mHideNewPostsBarOnScrollOffset) {
```

Comment:
Wouldn't this only work if you scrolled down? It should probably check for a delta of either positive or negative, so it works if you scroll up or down.